src/ocean/util/Convert.d

```diff
@@ -916,7 +916,8 @@ D toStringFromString(D,S)(S value)
     static if (is(S : D))
         return value;
 
-    else static if (!isMutable!(DElem) || !isMutable!(SElem))
```

Comment:
`isMutable` still looks better TBH

Reply:
True, what were the reasons to drop it?

Reply:
You reckon? Personally I don't like the double negative (if it is not not const and not not immutable ...) But the point is, the use of `isMutable`  is a blocker to removing ocean.transition, it doesn't exist elsewhere in ocean.

Reply:
Maybe `isMutable` should be made a type traits then ?
Or use a different condition, such as `is(Unqual!T == T)` to show that only unqualified types are accepted ?

Reply:
I do not think anything like that is justified for a single expression which is only used in one place in the internals of a function. I am trying to make the absolutely minimal changes required to finish the ocean D2 transition. I can't justify anything more complicated than that.